multi_tenant_test.go

```diff
@@ -196,7 +197,7 @@ func testMultiTenantByAppType(t *testing.T, appType string) {
 	require.ErrorContains(t, err, "connection refused")
 	require.Nil(t, resp)
 	if err == nil {
-		defer resp.Body.Close()
+		defer func() { _ = resp.Body.Close() }()
```

Comment:
might want to use `IgnoreErr` func that you introduced in `testhelper` pkg.

Reply:
In this case I'm getting a complaint by golangci-lint because it can't detect that we're actually closing the body. I will get rid of this helper.